lib/multigw_minifier.js

```diff
@@ -0,0 +1,207 @@
+require('apollojs');
+
+var Prefix = require('./prefix');
+var Db = require('./db');
+
+var kNothing = 0;
+var kPullLeft = 1;
+var kPullRight = 2;
+var kPullBoth = kPullLeft | kPullRight;
+var kPullData = 8;
+var kPullNothing = 16;
+
+var nColor = 0;
+var colorToNet = {};
```

Comment:
nColor and colorToNet should be passed around as a parameter for dfs() instead of a module variable.